### DISCUSSION TITLE
Selecting individual face of a component in imported assembly

### DISCUSSION THREAD

**Discussion by Somayaji1995** [ansys/pyaedt#4954] [Q&A] [answered]
I have an assembly which is imported using pyAEDT, as one of my component is inclined i need to create new coordinate system on its face and assign material properties for the same. In order to do that using script i have to call individual face names separately. Is there a way where in i can find face names of imported component using script?

**Comment by fwlfHS** [comment, marked as answer]
Do you know the names of the objects that have been imported?
If so you can use `obj = hfss.modeler.get_object_from_name('my_object')` and get access to the face ids of the object by `obj.faces`
